.github/workflows/publish.yml

```diff
@@ -26,8 +26,9 @@ jobs:
     - name: List result
       run: ls -l dist
 
+    # Only twine>=3.3 has --strict
```

Comment:
maybe instead if the comment here, add the 3.3 requirement to L21? (I cannot review suggest as that line is greyed out

Reply:
Oh yeah! That would make much more sense. Thanks!